section/storage.html

```diff
@@ -1,23 +1,167 @@
+        </p>
+        
+        <section data-dfn-for="Profile">
+            <h4>Profile</h4>
+            
+            <p>The {{Profile}} dictionary represents the complete user state for Web Monetization, containing wallet information, authentication credentials, active grants, and payment preferences.</p>
+            
+            <pre class="idl">
+            dictionary Profile {
+                // Wallet Information
+                WalletAddressDetails? wallet;
+                
+                // Authentication Keys
+                AuthKeys? keys;
+                
+                // Active Grant Information  
+                Grant? grant;
+                
+                // Payment Rate Information
+                PaymentRateInfo? rateInfo;
+            };
+            </pre>
+            
+            <dl>
+                <dt><dfn>wallet</dfn></dt>
+                <dd>Contains the user's wallet address information and server endpoints.<br><span class="note">Set when a wallet address is successfully retrieved and validated.</span></dd>
+                
+                <dt><dfn>keys</dfn></dt>
+                <dd>Contains the cryptographic key pair used for HTTP message signatures.<br><span class="note">Generated during initial setup and persisted for subsequent requests.</span></dd>
+                
+                <dt><dfn>grant</dfn></dt>
+                <dd>Contains the active grant tokens and management URLs.<br><span class="note">Set when a grant request is successfully completed and approved.</span></dd>
+                
+                <dt><dfn>rateInfo</dfn></dt>
+                <dd>Contains payment rate configuration for the current monetization session.<br><span class="note">This information is configured by the user.</span></dd>
+            </dl>
+        </section>
+        
+        <section data-dfn-for="AuthKeys">
+            <h4>Authentication Keys</h4>
+            
+            <p>The {{AuthKeys}} dictionary contains the cryptographic key pair used for HTTP message signatures in Open Payments requests.</p>
+            
+            <pre class="idl">
+            dictionary AuthKeys {
+                required DOMString keyId;
+                required DOMString privateKey;
+                required DOMString publicKey;
+            };
+            </pre>
+            
+            <dl>
+                <dt><dfn>keyId</dfn></dt>
+                <dd>The key identifier used in HTTP signature headers to reference this specific key pair.</dd>
+                
+                <dt><dfn>privateKey</dfn></dt>
+                <dd>The private key for signing HTTP messages. Must be stored securely and encrypted.</dd>
+                
+                <dt><dfn>publicKey</dfn></dt>
+                <dd>The corresponding public key that can be shared with servers for signature verification.</dd>
```

Comment:
We've already defined these in the "Ed25519 keypair" section. Should cross-reference those and define in single place.